plugins/generator-1.20.1/forge-1.20.1/templates/gui/gui_window.java.ftl

```diff
@@ -116,20 +116,20 @@ public class ${name}Screen extends AbstractContainerScreen<${name}Menu> {
 		</#list>
 	}
 
-	@Override protected void renderBg(PoseStack ms, float partialTicks, int gx, int gy) {
+	@Override protected void renderBg(GuiGraphics guiGraphics, float partialTicks, int gx, int gy) {
 		RenderSystem.setShaderColor(1, 1, 1, 1);
 		RenderSystem.enableBlend();
 		RenderSystem.defaultBlendFunc();
 
 		<#if data.renderBgLayer>
 			RenderSystem.setShaderTexture(0, texture);

```

Comment:
Here for example you still call setShaderTexture. Are you sure it is needed here?